### PR TITLE
added dispute endpoints

### PR DESCRIPTION
**Pull request by ankitdas13**
Api doc reference

[Fetch All Disputes](https://razorpay.com/docs/api/disputes/fetch-all)
[Fetch a Dispute](https://razorpay.com/docs/api/disputes/fetch)
[Accept a Dispute](https://razorpay.com/docs/api/disputes/accept)
[Contest a Dispute](https://razorpay.com/docs/api/disputes/contest)

### CODE REVIEW

File: documents/dispute.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    "access_activity_log": null,
+    "refund_cancellation_policy": null,
+    "term_and_conditions": null,
+    "others": null,
+    "submitted_at": null
+  }
+}
+```
+-------------------------------------------------------------------------------------------------------
+### Accpet a Dispute
```
Comment (shwatang) [inline documents/dispute.md:96]: spelling check

File: documents/dispute.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
+## Dispute
```
Comment (shwatang) [inline documents/dispute.md:1]: Please check for all comments here as added in this PR - https://github.com/razorpay/razorpay-dot-net/pull/122/files.

**Review by shwatang — COMMENTED** [review]
Have added minor comments, please check.

File: documents/dispute.md (hunk 1 above)
Comment (ankitdas13) [reply documents/dispute.md]: fix typo : https://github.com/razorpay/razorpay-dot-net/blob/added_dispute/documents/dispute.md?plain=1#L96

File: documents/dispute.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+billing_proof.Add("doc_EFtmUsbwpXwBH8");
+disputeRequest.Add("billing_proof", billing_proof);
+disputeRequest.Add("action", "submit");
+
+Dispute dispute = client.Dispute.Fetch(disputeId).Contest(disputeRequest);
+```
+
+**Response:**
+```json
+// Draft
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/dispute.md:197]: There are different responses depending on payload.

File: documents/dispute.md (hunk 2 above)
Comment (ankitdas13) [reply documents/dispute.md:1]: made changes

Comment (shwatang) [reply documents/dispute.md:1]: I see we have missed comments.

**Review by shwatang — APPROVED** [review]
LGTM. Please test the changes before merging.

**Pull request merged**